index.html

```diff
@@ -0,0 +1,311 @@
+            automatically convert it to <code>.html</code>, <code>.pdf</code>, or <code>.docx</code>,
+            and deploy it to your <a href="https://pages.github.com/">destination</a> of choice.
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="special_section">
+      <div class="section_header">How It Works</div>
+      <figure class="row flex_wrap workflow_row">
+        <figure class="workflow_figure">
+          <div class="workflow_image"></div>
+          <figcaption>input <code>markdown</code></figcaption>
+        </figure>
+        <div class="workflow_arrow">
+          <i class="fas fa-long-arrow-alt-right fa-lg"></i>
+        </div>
+        <figure class="workflow_figure">
+          <div class="workflow_image"></div>
+          <figcaption>
+            process w/ Manubot tools and <a href="https://pandoc.org/">Pandoc</a>
+          </figcaption>
+        </figure>
+        <div class="workflow_arrow">
+          <i class="fas fa-long-arrow-alt-right fa-lg"></i>
+        </div>
+        <figure class="workflow_figure">
+          <div class="workflow_image"></div>
+          <figcaption>
+            output <code>.html</code>, <code>.pdf</code>, <code>.docx</code>
+          </figcaption>
+        </figure>
+        <div class="workflow_arrow">
+          <i class="fas fa-long-arrow-alt-right fa-lg"></i>
+        </div>
+        <figure class="workflow_figure">
+          <div class="workflow_image"></div>
+          <figcaption>test, deploy to host</figcaption>
+        </figure>
+      </div>
+    </section>
+    <section>
+      <div class="section_header">The Best Parts</div>
+      <div class="row flex_wrap_reverse">
+        <div class="col_text">
+          <div class="col_header">
+            <i class="fas fa-atlas fa-lg"></i> Open, collaborative, versioned
+          </div>
+          <div>
+            Author your manuscript the open-source way.
+            Let anyone propose changes.
+            Iteratively review, discuss, and refine with collaborators.
+            Coordinate teams of any size with ease.
+            Keep track of versions, to-dos, revisions, and who wrote what and when.
+            All in a <a href="https://github.com/">central location</a>.
+          </div>
+        </div>
+        <div class="col_image"></div>
+      </div>
+      <div class="row flex_wrap">
+        <div class="col_image"></div>
+        <div class="col_text">
+          <div class="col_header">
+            <i class="fas fa-magic fa-lg"></i> Automated and continuous
+          </div>
+          <div>
+            Never use a reference manager again.
+            Provide a DOI, ISBN, PubMed ID, URL, etc., and let Manubot generate and number citations for you.
+            Choose from <a href="https://www.zotero.org/styles">thousands of existing citation styles</a>.
+            See your manuscript update right away as you make changes.
```

Comment:
To me, this implies a greater level of interactivity that what we can currently offer.  I would think of the Overleaf interface when I read "right away", in which the PDF output can update within seconds of changing source material.

Reply:
I think "right away" is vague enough to apply to how Manubot works currently. I considered writing "instantly", but we don't have hot-reloading yet. But I think we do eventually want to have that as a high priority, such that you can have the autobuild script running and also have chrome/firefox automatically refresh the tab.

Even though the build process might take a couple of seconds (with cached refs), I don't think that should prevent us from saying "right away".